public/src/components/channelManagement/headerTests/headerTestEditor.tsx

```diff
@@ -20,55 +19,47 @@ import TestEditorVariantSummary from '../testEditorVariantSummary';
 import { ControlProportionSettings } from '../helpers/controlProportionSettings';
 import TestVariantsSplitEditor from '../testVariantsSplitEditor';
 import { useStyles } from '../helpers/testEditorStyles';
+import { TestEditorProps } from '../testsForm';
 
-interface HeaderTestEditorProps {
-  test: HeaderTest;
-  onChange: (updatedTest: HeaderTest) => void;
-  onValidationChange: (isValid: boolean) => void;
-  visible: boolean;
-  editMode: boolean;
-  onDelete: () => void;
-  onArchive: () => void;
-  onTestSelected: (testName: string) => void;
-  testNames: string[];
-  testNicknames: string[];
-  createTest: (newTest: HeaderTest) => void;
-}
-
-const HeaderTestEditor: React.FC<HeaderTestEditorProps> = ({
+const HeaderTestEditor: React.FC<TestEditorProps<HeaderTest>> = ({
```

Comment:
Most of this file hasn't changed. It now takes the standard `TestEditorProps` and renders the `StickyTopBar`